Mathlib/RingTheory/LocalRing/RingHom/Basic.lean

```diff
@@ -116,6 +116,14 @@ theorem of_surjective [CommSemiring R] [IsLocalRing R] [CommSemiring S] [Nontriv
       (isUnit_or_isUnit_of_isUnit_add <| IsLocalHom.map_nonunit _ hab).imp f.isUnit_map
         f.isUnit_map)
 
+lemma _root_.IsLocalHom.of_surjective [CommRing R] [CommRing S] [Nontrivial S] [IsLocalRing R]
```

Comment:
Or maybe `Function.Surjective.isLocalHom`?
```suggestion
lemma _root_.Function.Surjective.isLocalHom [CommRing R] [CommRing S] [Nontrivial S] [IsLocalRing R]
```
What do you think?

Reply:
I added an alias.